bundles/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/internal/handler/HDPowerViewRepeaterHandler.java

```diff
@@ -0,0 +1,218 @@
+                    repeaterData = webTargets.enableRepeaterBlinking(repeaterId, OnOffType.ON == command);
+                    scheduler.submit(() -> updatePropertyAndState(repeaterData));
+                    break;
+            }
+        } catch (HubInvalidResponseException e) {
+            Throwable cause = e.getCause();
+            if (cause == null) {
+                logger.warn("Bridge returned a bad JSON response: {}", e.getMessage());
+            } else {
+                logger.warn("Bridge returned a bad JSON response: {} -> {}", e.getMessage(), cause.getMessage());
+            }
+        } catch (HubMaintenanceException e) {
+            // exceptions are logged in HDPowerViewWebTargets
+        } catch (HubException e) {
+            logger.warn("Unexpected error: {}", e.getMessage());
+        }
+    }
+
+    private void cancelResetIdentifyStateJob() {
+        ScheduledFuture<?> scheduledJob = resetIdentifyStateFuture;
+        if (scheduledJob != null) {
+            scheduledJob.cancel(true);
+        }
+        resetIdentifyStateFuture = null;
+    }
+
+    private void scheduleRefreshJob() {
+        cancelRefreshJob();
+        logger.debug("Scheduling poll for repeater {} now, then every {} minutes", repeaterId,
+                REFRESH_INTERVAL_MINUTES);
+        this.refreshStatusFuture = scheduler.scheduleWithFixedDelay(this::poll, 0, REFRESH_INTERVAL_MINUTES,
+                TimeUnit.MINUTES);
+    }
+
+    private void cancelRefreshJob() {
+        ScheduledFuture<?> future = this.refreshStatusFuture;
+        if (future != null) {
+            future.cancel(false);
+        }
+        this.refreshStatusFuture = null;
+    }
+
+    private synchronized void poll() {
+        HDPowerViewHubHandler bridge = getBridgeHandler();
+        if (bridge == null) {
+            logger.warn("Missing bridge handler");
+            return;
+        }
+        HDPowerViewWebTargets webTargets = bridge.getWebTargets();
+        if (webTargets == null) {
+            logger.warn("Web targets not initialized");
+            return;
+        }
+        try {
+            logger.debug("Polling for status information");
+
+            RepeaterData repeaterData = webTargets.getRepeater(repeaterId);
+            updatePropertyAndState(repeaterData);
+
+        } catch (HubInvalidResponseException e) {
+            Throwable cause = e.getCause();
+            if (cause == null) {
+                logger.warn("Bridge returned a bad JSON response: {}", e.getMessage());
+            } else {
+                logger.warn("Bridge returned a bad JSON response: {} -> {}", e.getMessage(), cause.getMessage());
+            }
+        } catch (HubMaintenanceException e) {
+            // exceptions are logged in HDPowerViewWebTargets
+        } catch (HubException e) {
+            logger.warn("Error connecting to bridge: {}", e.getMessage());
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_OFFLINE, e.getMessage());
```

Comment:
The log message could be removed, as the status update is already logged by the framework.

The state change originated by updateStatus() is logged to events.log. Including the status detail message.

Reply:
Good point, removed.